templates/secrets/metadata-connection-secret.yaml

```diff
@@ -2,7 +2,12 @@
 ## Airflow Metadata Secret
 #################################
 {{- if (and .Values.data.metadataConnection (not .Values.data.metadataSecretName)) }}
-{{- $host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $postgres_host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $pgbouncer_host := (printf "%s-%s.%s.svc.cluster.local" .Release.Name "pgbouncer" .Release.Namespace) }}
```

Comment:
@dimberman small nit, but I think most values and variables are camelCase in the chart now.

Reply:
@schnie fixed :)